spec/controllers/api/items_controller_spec.rb

```diff
@@ -245,27 +246,229 @@
             expect(saved_items[3]).to match(items_param[3])
           end
         end
+
+        context 'and syncing items without the items_key_id field' do
+          it 'should set a default value for the field' do
+            @controller = Api::AuthController.new
+            post :sign_in, params: test_user_003_credentials
+
+            @controller = Api::ItemsController.new
+            request.headers['Authorization'] = "bearer #{JSON.parse(response.body)['token']}"
+
+            new_item = create(:item, :with_items_key_id, user_uuid: test_user.uuid)
+
+            items_param = [new_item].to_a.map(&:serializable_hash)
+            items_param[0].delete('items_key_id')
+
+            post :sync, params: { limit: 5, api: '20190520', items: items_param }
+
+            expect(response).to have_http_status(:ok)
+            expect(response.headers['Content-Type']).to eq('application/json; charset=utf-8')
+
+            parsed_response_body = JSON.parse(response.body)
+
+            expect(parsed_response_body).to_not be_nil
```

Comment:
Same here. This test shouldn't be testing for this, as it should have been tested in other tests.